src/main/java/net/hypixel/nerdbot/generator/MinecraftImage.java

```diff
@@ -0,0 +1,184 @@
+package net.hypixel.nerdbot.generator;
+
+import net.hypixel.nerdbot.command.ItemGenCommand;
+import net.hypixel.nerdbot.util.skyblock.MCColor;
+import org.jetbrains.annotations.Nullable;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+
+public class MinecraftImage {
+    int locationX = 10;
+    int locationY = 25;
+
+    GraphicsEnvironment ge;
+    Graphics2D g2d;
+    final Font minecraftFont;
+    final Font minecraftBold;
+    MCColor currentColor;
+    boolean boldFlag = false; //True if we're currently printing with bold, false if not.
+    private BufferedImage image;
+
+    public MinecraftImage(int imageWidth, int linesToPrint, MCColor defaultColor) {
+        this.ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        this.minecraftFont = initFont("/Minecraft/minecraft.ttf", 16f);
+        this.minecraftBold = initFont("/Minecraft/3_Minecraft-Bold.otf", 22f);
+        this.g2d = initG2D(imageWidth, linesToPrint * 23);
+        this.currentColor = defaultColor;
+    }
+
+    public MinecraftImage(int imageWidth, int linesToPrint) {
+        this.ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        this.minecraftFont = initFont("/Minecraft/minecraft.ttf", 16f);
+        this.minecraftBold = initFont("/Minecraft/3_Minecraft-Bold.otf", 22f);
+        this.g2d = initG2D(imageWidth, linesToPrint * 23);
+        this.currentColor = MCColor.GRAY;
+    }
```

Comment:
Couldn't we use the other constructor and set the third parameter to `MCColor.GRAY`?

Reply:
Yeah sure that's ez